blockstore_proxy.go

```diff
@@ -43,14 +43,20 @@ func (ps *proxyBlockStore) Fetch(ctx context.Context, path string, cb lib.DataCa
 		return err
 	}
 	goLog.Debugw("car fetch", "url", req.URL)
-	req.Header.Set("Accept", "application/vnd.ipld.car")
+	req.Header.Set("Accept", "application/vnd.ipld.car;order=dfs;dups=y")
```

Comment:
Note: this resolves #177 for non-Saturn backends. Caboose/Saturn will have to change when they're ready.